.circleci/config.yml

```diff
@@ -71,6 +71,21 @@ workflows:
       - build-and-test/build-and-test:
           name: build-and-test-postgres-wal-json
           script: ./scripts/build-and-test-postgres-wal-json.sh
+      - build-and-test/build-and-test:
```

Comment:
This is an extremely expensive way to test a low-level feature. Every additional job adds minutes to the build:

0. Wait for an available VM
1. Create VM to run the job
2. Restore cache
3. compile lots of code
4. Run one test, e.g. io.eventuate.local.polling.PollingBinlogEntryReaderMessageTableTest
5. shut everything down

A more efficient approach would be a single job that perhaps runs first and starts all of the databases and runs the needed tests.

Reply:
ok